serenata_toolbox/datasets/__init__.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from serenata_toolbox.datasets.downloader import Downloader
+from serenata_toolbox.datasets.local import LocalDatasets
+from serenata_toolbox.datasets.remote import RemoteDatasets
+
+
+class Datasets:
+    """
+    This is a wrapper for three different classes together handles the datasets
```

Comment:
you got me there! I'm not sure what is the best approach on the matter. I new to the whole doctstrings idea =|